rasa/core/policies/_policy.py

```diff
@@ -428,3 +428,139 @@ def format_tracker_states(states: List[Dict]) -> Text:
                     formatted_states.append(state_formatted)
 
         return "\n".join(formatted_states)
+
+
+class PolicyPrediction:
```

Comment:
it's a copy of the previous policy prediction - in the new one (`PolicyPrediction2`), priorities are just ints (not tuples containing 1 int) and the max. probability is not called "max. confidence"

Reply:
revert or keep?

Reply:
Priorities are also one `int` [here](https://github.com/RasaHQ/rasa/blob/19fb70a05e3637efb9512a942c56cd12ad2c738e/rasa/core/policies/policy.py#L511), aren't they?

I like the rename. The only issue is that `PolicyPrediction` might be used by users who have custom policies. Means we'd have to make sure to add this change to the migration guide or make this non breaking

Reply:
just reverted it

Reply:
oh forgot to reload and didn't see your comment :D  -- I doubt that users access the internals of PolicyPredictions (they'd need to write custom ensembles for that) because the constructor does not change -- but Daksh reminded me that it might be used elsewhere